src/modules/fancyzones/lib/Settings.cpp

```diff
@@ -52,6 +53,11 @@ struct FancyZonesSettings : winrt::implements<FancyZonesSettings, IFancyZonesSet
     const std::wstring m_zoneHighlightOpacity = L"fancyzones_highlight_opacity";
 };
 
+bool FancyZonesSettings::Init(PCWSTR config, bool fromFile) noexcept

```

Comment:
Not a part of this work but we should probably rename the `config` are to something more descriptional.. Same for `m_name`